_layouts/person.html

```diff
@@ -22,11 +22,11 @@ <h2>{{ page.role | escape}}</h2>
           {{ page.phone }}</br>
           {{ page.room }}</br>
           {% if page.google_map != null %}
-          <a href="{{ page.google_map }}" target=_blank>
+          <a href="{{ page.google_map }}" target=_blank><img class="cnerg-service-icon" src="{{ site.baseurl }}/services/icons/google_maps.png">
```

Comment:
Is baseurl required? Usually absolute paths without domain resolve properly.

I expect scaling will be necessary. Have you tried rendering this yet?

Reply:
Not sure why I put the baseurl in for the other service icons, but I know it works.

Scaling is handled by the CSS class.